tests/integration/schema_registry/test_json_serializers.py

```diff
@@ -0,0 +1,188 @@
+import pytest
+from confluent_kafka.cimpl import TopicPartition
+
+from confluent_kafka.schema_registry.json_schema import JsonSerializer, JsonDeserializer
+from confluent_kafka.serialization import SerializationError
+
+
+class _TestProduct(object):
+    def __init__(self, product_id, name, price, tags, dimensions, location):
+        self.product_id = product_id
+        self.name = name
+        self.price = price
+        self.tags = tags
+        self.dimensions = dimensions
+        self.location = location
+
+
+def _testProduct_to_dict(ctx, product_obj):
+    """
+    Returns testProduct instance in dict format.
+
+    Args:
+        ctx (SerializationContext): Metadata pertaining to the serialization
+                operation.
+
+        product_obj (_TestProduct): testProduct instance.
+
+    Returns:
+        dict: product_obj as a dictionary.
+
+    """
+    return {"productId": product_obj.product_id,
+            "productName": product_obj.name,
+            "price": product_obj.price,
+            "tags": product_obj.tags,
+            "dimensions": product_obj.dimensions,
+            "warehouseLocation": product_obj.location}
+
+
+def _testProduct_from_dict(ctx, product_dict):
+    """
+    Returns testProduct instance from it's dict format.
+
+    Args:
+        ctx (SerializationContext): Metadata pertaining to the serialization
+                operation.
+
+        product_dict (dict): testProduct in dict format.
+
+    Returns:
+        _TestProduct: product_obj instance.
+
+    """
+    return _TestProduct(product_dict['productId'],
+                        product_dict['productName'],
+                        product_dict['price'],
+                        product_dict['tags'],
+                        product_dict['dimensions'],
+                        product_dict['warehouseLocation'])
+
+
+def test_json_record_serialization(kafka_cluster, load_file):
+    """
+    Tests basic JsonSerializer and JsonDeserializer basic functionality
+
+    product.json from:
+        https://json-schema.org/learn/getting-started-step-by-step.html
+
+    Args:
+        kafka_cluster (KafkaClusterFixture): cluster fixture
+
+        load_file (callable(str)): JSON Schema file reader
+
+    """
+    topic = kafka_cluster.create_topic("serialization-json")
+    sr = kafka_cluster.schema_registry({'url': 'http://localhost:8081'})
```

Comment:
Why is the URL specified here? I thought trivup's SR allocates a port?
Having fixed port numbers should be avoided since it will collide with other instances, programs, etc.

Reply:
Accidentally left it. It's there as an override for an external SR since CP 5.5 isn't out yet.

Reply:
on the bright side look how easy it is to use an external SR instance with trivup